tests/test_md_cli.py

```diff
@@ -102,7 +105,9 @@ def test_log(tmp_path):
 
     assert_log_contains(log_path, includes=["Starting molecular dynamics simulation"])
 
-    with open(tmp_path / "nvt-T300-stats.dat", encoding="utf8") as stats_file:
+    with open(
+        tmp_path / "nvt-T300-nvt-T300.0-stats.dat", encoding="utf8"
```

Comment:
Is this supposed to have `nvt` twice? (also in the `stats_path`)

Reply:
yes since one comes from the prefix bits. is ugly but is just a test.